compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1292,7 +1301,8 @@ TR::Register *OMR::X86::TreeEvaluator::SSE2ArraycmpLenEvaluator(TR::Node *node,
 
    TR::Register *s1Reg = cg->gprClobberEvaluate(s1AddrNode, TR::InstOpCode::MOVRegReg());
    TR::Register *s2Reg = cg->gprClobberEvaluate(s2AddrNode, TR::InstOpCode::MOVRegReg());
-   TR::Register *strLenReg = cg->gprClobberEvaluate(lengthNode, TR::InstOpCode::MOVRegReg());
+   TR::Register *strLenReg = cg->gprClobberEvaluate(lengthNode, TR::InstOpCode::MOV8RegReg);
```

Comment:
I doubt that `MOV8RegReg` works on 32-bit. Please use `longClobberEvaluate()`. On 64-bit it will do this, but on 32-bit it will use a pair of `MOV4RegReg` instructions instead

Reply:
`longClobberEvaluate` is the functionality I wanted, thanks for bringing it up